openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -142,10 +145,11 @@ def test_calculate_course_xblocks_data(self, course_attr):
                     )
 
     @ddt.data(
-        ('course', 47),
-        ('other_course', 34)
+        ('course', 48),
+        ('other_course', 35)
     )
     @ddt.unpack
+    @skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Different query counts in CMS')
```

Comment:
I tried debugging this some-- when run in CMS, disabled_xblocks is already stored in the request cache when the test point runs (so something in setup causes it to get stored). When run from LMS, that isn't the case. I tried going through the setup code but can't figure out why the CMS vs LMS behavior differs.